### ISSUE TITLE
8.0.2 postcss update has bundling issue?

### ISSUE THREAD

**Issue by jkinkead** [postcss/postcss-scss#111]
We're seeing errors in our build using the latest `postcss-scss` version, I think because of the bundling issue mentioned in [8.0.3](https://github.com/postcss/postcss/releases/tag/8.0.3)?

The error is:
```
TypeError: Class constructor Parser cannot be invoked without 'new'
    at new ScssParser (node_modules/postcss-scss/lib/scss-parser.js:21:53)
```

**Comment by ai** [comment]
Can you update `postcss-scss` and `postcss` to the latest version?

If you still have a config, show what are you doing before getting an error. What config do you have? What did you run?

**Comment by jkinkead** [comment]
Unfortunately, this is being run via a GitHub formatting action (`prettier` via `pre-commit`) and I don't have access to the runner box, and I can't reproduce locally.

**Comment by ai** [comment]
On line 21 we do not have `new` right now.

Seems like you are using old `postcss-scss` 2.x, which do not support PostCSS 8.

I am closing the issue. Feel free to reopen if we will have more details.

**Comment by jkinkead** [comment]
It's my understanding that the error comes from trying to use `extends` on a transpiled class.

This is fixed in [postcss #1412](https://github.com/postcss/postcss/issues/1412); sorry I couldn't be of more help in pinning this down!
